apps/web/src/modules/dashboard/DaoAuctionCard.tsx

```diff
@@ -0,0 +1,159 @@
+import { Box, Flex, Text } from '@zoralabs/zord'
+import dayjs from 'dayjs'
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+import React, { useState } from 'react'
+import { formatEther } from 'viem'
+import { useContractEvent } from 'wagmi'
+
+import { PUBLIC_ALL_CHAINS } from 'src/constants/defaultChains'
+import { auctionAbi } from 'src/data/contract/abis'
+import { useCountdown, useIsMounted } from 'src/hooks'
+import { AddressType } from 'src/typings'
+
+import { AuctionPaused } from './AuctionPaused'
+import { BidActionButton } from './BidActionButton'
+import { DashboardDao } from './Dashboard'
+import {
+  auctionCardBrand,
+  bidBox,
+  daoAvatar,
+  daoAvatarBox,
+  daoTokenName,
+  outerAuctionCard,
+  stats,
+  statsBox,
+} from './dashboard.css'
+
+export const DaoAuctionCard = (
```

Comment:
can we separate out the type definition here for readability, would also be nice to destructure the types in the parameters for readability -- also can we rename `DashboardDao` to `DashboardDaoProps` or something similar so we know its a type definition